src/JabbR-Core/Hubs/Chat.cs

```diff
@@ -186,7 +187,7 @@ public void UpdateActivity()
                 UpdateActivity(user, room);
             }
 
-            CheckStatus();
+            CheckStatus(testing);
```

Comment:
The `CheckStatus` method checks versions, but the `Context.QueryString` object is null, and read only. I haven't found a way to spoof this yet, but the version checks are non-critical for testing so I just allowed calls to Send with an optional testing parameter that would skip over the test. Still working on this.